### PR TITLE
Add vic appliance ip information in vic plugin summary view. 

### PR DESCRIPTION
**Pull request by zhoumeina**
As the user is not able to select which API server to use (#572), providing this information may greatly aid debugging when multiple versions of API servers are available.
Fix:
1 Add vic appliance version information in summary view. 
2 Add vic appliance ip address in summary view.

### CODE REVIEW

**Comment by zhoumeina** [comment]
image

File: h5c/vic/src/vic-webapp/src/app/summary-view/summary-view.component.ts [hunk 1]
```diff
@@ -96,6 +97,7 @@ export class VicSummaryViewComponent implements OnInit, OnDestroy {
       this.createWzService.verifyVicMachineApiEndpoint()
       .subscribe(
         (ip: string) => {
+          this.applianceIp = ip;
```
Comment (zjs) [inline h5c/vic/src/vic-webapp/src/app/summary-view/summary-view.component.ts]: It seems like we only set the IP if an error does not occur. Given the goal of aiding a user in debugging, it seems like we would also want to set the IP in that case too.

**Comment by javierfz1980** [comment]
There are some comments,

Also please provide the `acceptance criteria` checklist:

--------------------

Fixes #572  (the number of the issue this PR is fixing/adding)

PR acceptance checklist:

[ ] All unit tests pass
[ ] All e2e tests pass
[ ] Unit test(s) included*
[ ] e2e test(s) included*
[ ] Screenshot attached and UX approved*

*if applicable, add n/a if not

------------------------

**Comment by renmaosheng** [comment]
please update the commit message as well, given that ova version is also added.

**Pull request merged**